usaspending_api/disaster/v2/views/federal_account/spending.py

```diff
@@ -31,16 +34,41 @@ def construct_response(results: list, pagination: Pagination):
     }
 
 
-class SpendingViewSet(PaginationMixin, SpendingMixin, FabaOutlayMixin, DisasterBase):
+class SpendingViewSet(
+    PaginationMixin,
+    SpendingMixin,
+    FabaOutlayMixin,
+    ElasticsearchAccountDisasterBase,
+    ElasticsearchSpendingPaginationMixin,
+    DisasterBase,
+):
     """ Returns disaster spending by federal account. """
 
     endpoint_doc = "usaspending_api/api_contracts/contracts/v2/disaster/federal_account/spending.md"
+    agg_key = "financial_accounts_by_award.treasury_account_id"  # "financial_accounts_by_award.federal_account_id"  # primary (tier-1) aggregation key
```

Comment:
Same as previous comment about `federal_account_id` or `treasury_account_id` being the `agg_key` vs. `sub_agg_key`